R/UK.R

```diff
@@ -141,7 +143,7 @@ UK <- R6::R6Class("UK", # rename to country name
     #' @importFrom rlang .data
     clean_level_2 = function() {
       self$get_authority_lookup_table()
-      self$data$clean <- self$data$raw %>%
+      self$data$clean <- self$data$raw[[1]] %>%
```

Comment:
probably a good idea to ask contributors to assign some kind of standard name?

Reply:
the names should get rendered from the filters, I just used 1 here to get the first element for level 1 but happy to change this to nation / region

Reply:
*utla